src/compressed_tensors/utils/helpers.py

```diff
@@ -35,11 +38,85 @@ def infer_compressor_from_model_config(
     :return: matching compressor if config contains a sparsity config
     """
     config = AutoConfig.from_pretrained(pretrained_model_name_or_path)
-    sparsity_config = getattr(config, SPARSITY_CONFIG_NAME, None)
+    sparsity_config = getattr(config, CONFIG_NAME, None)
     if sparsity_config is None:
         return None
 
     format = sparsity_config.get("format")
     sparsity_config = CompressionConfig.load_from_registry(format, **sparsity_config)
     compressor = ModelCompressor.load_from_registry(format, config=sparsity_config)
     return compressor
+
+
+def save_compressed(
```

Comment:
Yes, this is my plan. @bfineran proposed to create some `Mixin` class that exposes those functions as class methods, so that `SparseAutoModel(AutoModel, Mixin)`